scss/sin-core/grid/_simple-grid-mixin.scss

```diff
@@ -9,19 +9,21 @@
 /// @param  {number}  $template-columns [12] - Breakpoint will work up to `$max`
 /// @content
 /// @example scss A layout with three columns
-/// .grid-example{
-///   @include simple-grid(3){
-///     background-color: purple;
-///     padding: 1rem;
+///   @use 'grid/simple-grid-mixin' as *;
```

Comment:
This was the main issue -- the code inside an `@example` needs to do its own imports (`@use` or `@include`), or else the imports need to be handled automatically in the `.sassdocrc` config.

Reply:
As I said above, probably my `.sassdocrc` `use:` was wrong. Now I have individual `@use` inside each example, so this part has been resolved!